utils.py

```diff
@@ -1,34 +1,63 @@
 import os
+import subprocess
 from pathlib import Path
+import shutil
 import platform
+import sys
 
 system = platform.system()
 
-def get_nerts_path():
-  # Try to find where Nerts is installed
-  if system == "Darwin":
-    nertsPath = Path(os.path.join(os.path.expanduser('~'), "Library/Application Support/Steam/steamapps/common/Nerts Online/NERTS! Online.app/Contents/MacOS/NertsOnline.exe"))
-  elif system == "Windows":
-    nertsPath = Path("C:/Program Files (x86)/Steam/steamapps/common/Nerts Online/NERTS! Online.exe")
 
-  while nertsPath is None or not nertsPath.exists():
+def remove_existing_path(path: Path) -> bool:
+    """
+    Removes a pathlib.Path if it exists.
+
+    returns:
+        - True on success,
+        - False on rmtree error
+    """
+
     try:
-      nertsPath = Path(input("Please enter the path to NertsOnline.exe: "))
-    except KeyboardInterrupt:
-      quit()
-    except:
-      print("")
-      pass
+        if path.exists():
+            shutil.rmtree(path)
+    except Exception:
+        return False
+
+    return True
+
+
+def get_nerts_path():
+    # Try to find where Nerts is installed
+    if system == "Darwin":
+        nertsPath = Path(os.path.join(os.path.expanduser('~'), "Library/Application Support/Steam/steamapps/common/Nerts Online/NERTS! Online.app/Contents/MacOS/NertsOnline.exe"))
+    elif system == "Windows":
+        nertsPath = Path("C:/Program Files (x86)/Steam/steamapps/common/Nerts Online/NERTS! Online.exe")
+
+    while nertsPath is None or not nertsPath.exists():
+        try:
+            nertsPath = Path(input("Please enter the path to NertsOnline.exe: "))
+        except KeyboardInterrupt:
+            sys.exit()
+        except Exception as e:
+            print(e)
+
+    return nertsPath
+
+
+def run_exe(path, args: list[str] = None, cwd=None):
+    command = str(path)
+
+    if args is None:
+        args = []
+
+    if cwd is None:
+        cwd = Path().cwd()
 
-  return nertsPath
+    command_list = [command] + args
 
-def run_exe(path, args = None):
-  command = str(path)
+    print(f"running {command_list} from {cwd}")
 
-  if not args is None:
-    command = command + " " + args
+    if system == "Darwin":
+        command_list.insert(0, "mono64")
 
-  if system == "Darwin":
-    os.system("mono64 " + command)
-  else:
-    os.system(command)
+    subprocess.call(command_list, cwd=cwd, shell=True)
```

Comment:
Unfortunately I don't think this works as you want it to on Windows. Reading [this](https://stackoverflow.com/questions/20128114/pythons-subprocess-does-not-interpret-as-expected-on-cygwin/20137299#20137299), it seems like the arguments get ignored:

> If args is a sequence, the first item specifies the command string, and any additional items will be treated as additional arguments to the shell itself.

We may just want to join them together if that works on Windows? Or find a different solution.